spec/features/course_pages_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe 'CoursePages', type: :feature do
```

Comment:
This should be a phrase, since it's not a class we're describing. It's kind of like a [user story](https://en.wikipedia.org/wiki/User_story).